sdk/eventgrid/README.md

```diff
@@ -0,0 +1,35 @@
+# Azure Event Grid libraries for .NET
+
+Azure Event Grid allows you to build applications with event-based architectures. The Event Grid service fully manages all routing of events from any source, to any destination, for any application. Azure service events and custom events can be published directly to the service, where the events can then be filtered and sent to various recipients, such as built-in handlers or custom webhooks. To learn more about Azure Event Grid: [What is Event Grid?](https://docs.microsoft.com/azure/event-grid/overview)
+
+## Libraries for data access
+
+The current generation of the Azure Event Grid client library uses versions 4.0 and above.  Microsoft recommends using version 4.2 or higher for new applications.  If you are unable to existing applications to version 4.x, then Microsoft recommends using version 3.2 or higher.
```

Comment:
Probably should avoid mentioning 4.2 as this will quickly become out of date and will be a pain to maintain.